### PR TITLE
Adds Dijkstra map add-on

### PR DESCRIPTION
**Pull request by astrale-sharp**
Hi there! Adds a new addon that i helped developping ;)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,6 +165,7 @@ See [Vivraan/godot-lang-support](https://github.com/Vivraan/godot-lang-support).
 - [Anima](https://github.com/ceceppa/anima) - Run sequential and parallel animations with less code compared to Tween (Godot 3.x).
 - [Aseprite Wizard](https://github.com/viniciusgerevini/godot-aseprite-wizard) - Plugin for importing animations from Aseprite as SpriteFrames (Godot 3.x).
 - [CRT Shader](https://github.com/hiulit/Godot-3-2D-CRT-Shader) - A Godot shader that simulates CRT Displays with many shader parameters (Godot 3.x).
+- [Dijkstra map for Godot](https://github.com/MatejSloboda/Dijkstra_map_for_Godot) - A GDNative project for Godot game engine, that introduces Dijktra Map pathfinding node.
```
Comment (Calinou) [inline README.md]: The new listing should be moved to the "Godot 3.2+" list below, since it most likely doesn't work with Godot 3.1 and older.

Comment (astrale-sharp) [reply README.md]: Yes, it makes more sense and i have committed and pulled the change

Comment (astrale-sharp) [reply README.md]: Was there any more thing to correct @Calinou ? (is it okay to ping?)

**Review by Calinou — APPROVED** [review]
Thanks! Congratulations for your first merged pull request :tada:

**Pull request merged**